mne/io/brainvision/brainvision.py

```diff
@@ -321,27 +323,32 @@ def _read_vmrk_events(fname, event_id=None, trig_shift_by_type=None):
             trigger = event_id[mdesc]
         else:
             try:
-                trigger = int(re.findall(r'[A-Za-z]*\s*?(\d+)', mdesc)[0])
+                trigger = int(re.findall(r'S([\s\d]{2}\d{1})', mdesc)[0])
```

Comment:
yyyyyeah. Is this intentional?

Reply:
yes, that's intentional - we are matching the BrainVision triggers, which are always in the format specified there.

"S" and then an up to 3 digit integer where the empty spaces are potentially padded by spaces. E.g., 1 would become `S  1` 

Am I missing an obvious flaw?

Reply:
Well, I do have a bunch of vmrks with no S ... they're from a lab using PyCorder iirc.

Reply:
So there was no conversion from some format into BrainVision that could be blamed? Would be great to know for sure :-) 

In any case, that problem is easily fixed by using the `event_id`. 

What this PR did was to adjust MNEs behavior to what it always said it would do (which is "read events that follow the S### pattern"). Are you voting on changing that?

One further information: With the BIDS-EEG team we have recently reached out to Brain Products and asked them to release a documentation of the BrainVision format to the community and they agreed, work is on the way. I suggest that we wait for that spec and then check how we can improve our BV reader.

Reply:
where did you see it broken?

Reply:
>where did you see it broken?

What do you mean?

Reply:
> Well, I do have a bunch of vmrks with no S ... they're from a lab using PyCorder iirc.

I mean do we have a test for that? or is it that we are missing the data?